css-nesting-1/Overview.bs

```diff
@@ -225,6 +225,20 @@ Direct Nesting {#direct}
 		   :is(.foo, .bar) + .baz,
 		   :is(.foo, .bar).qux { color: red; }
 		 */
+
+		ul, ol {
+			&:not(.foo,.bar) > .baz { color: red; }
+		}
+		/* equivalent to
+			 :is(ul,ol):not(.foo,.bar) > .baz { color: red; }
+		 */
+
+		.foo {
+			&:is(.bar, &.baz) { color: red; }
```

Comment:
Hmm, I'm not sure this (`&.baz`) is valid without `@nest`.

Reply:
I agree, I think an `@nest` should be required here

Reply:
No, that's fine. The only requirement is that the overall selector needs to start with an `&` (or else it'll instead be parsed as a (likely invalid) declaration. @nest lets you violate that rule and do whatever.

So in this case it's fine, the selector starts with an `&`.